### PR TITLE
feat(STONEINTG-838): update codeowners for konflux-ci team 

### PR DESCRIPTION
**Pull request by Josh-Everett**

### CODE REVIEW

**Comment by MartinBasti** [comment]
Codeowner file is invalid, GH shows warning in files view

Also infra-deployments repo has't been migrated the repo should not be changed

File: ci/promote-overlay/promote-overlay.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ fi
 UPDATE_BRANCH_NAME="integration-service-${TARGET_OVERLAY}-update-"$(date '+%Y_%m_%d__%H_%M_%S')
 
 # GitHub repository details
-owner="redhat-appstudio"
+owner="konflux-ci"
```
Comment (MartinBasti) [inline ci/promote-overlay/promote-overlay.sh]: This still relates to infra-deployments, konflux-ci/infra-deployments doesn't exist

Comment (hongweiliu17) [reply ci/promote-overlay/promote-overlay.sh]: We may update this `owner` in another PR after infra-deployments and integratio-service are migrated to `konflux-ci`.

Comment (Josh-Everett) [reply ci/promote-overlay/promote-overlay.sh]: moved related changes to a different [pr](https://github.com/konflux-ci/integration-service-utils/pull/5)

**Review by hongweiliu17 — APPROVED** [review]
LGTM

**Review by dirgim — APPROVED** [review]
LGTM

**Review by MartinBasti — APPROVED** [review]

**Comment by MartinBasti** [comment]
What did change that codeowners are valid now?

**Review by jsztuka — APPROVED** [review]

**Comment by Josh-Everett** [comment]
We needed to be moved into the konflux-ci org in order to reference the team from codeowners, once the org was transitioned the error was removed.

**Pull request merged**